src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/LicenceIntegrationTest.kt

```diff
@@ -585,16 +586,18 @@ class LicenceIntegrationTest : IntegrationTestBase() {
       .expectStatus().isOk
 
     val result = webTestClient.get()
-      .uri("/licence/id/1")
+      .uri("/events/match?licenceId=1&eventType=VARIATION_SUBMITTED_REASON")
       .accept(MediaType.APPLICATION_JSON)
```

Comment:
Checks that the reason was record in licence_events, not now on the licence itself.